stdlib/jdbc/src/main/java/org/ballerinax/jdbc/statement/UpdateStatement.java

```diff
@@ -74,24 +78,26 @@ public Object execute() {
             ArrayValue generatedParams = constructParameters(parameters);
             conn = getDatabaseConnection(strand, client, datasource);
             String processedQuery = createProcessedQueryString(query, generatedParams);
-            stmt = conn.prepareStatement(processedQuery, Statement.RETURN_GENERATED_KEYS);
+
+            boolean keyRetrievalSupportedStatement = isKeyRetrievalSupportedStatement();
```

Comment:
Shall we change this to something like below and use that boolean for other two places? That will avoid taking `&&` twice with the same booleans.

`boolean retrieveGeneratedKeys = isKeyRetrievalSupportedStatement() && getGeneratedKey`